.travis.yml

```diff
@@ -7,14 +7,14 @@ node_js:
   - '4'
 
 before_install: yarn global add greenkeeper-lockfile@1
+install: yarn install --ignore-engines
 
 before_script: greenkeeper-lockfile-update
 after_script: greenkeeper-lockfile-upload
 
 after_success:
   - bash <(curl -s https://codecov.io/bash)
-  - npm install -g semantic-release
```

Comment:
How about doing this instead:
```yaml
# ...

after_success:
  - yarn global add travis-deploy-once
  - travis-deploy-once "yarn global add semantic-release@12 && semantic-release"
```

Reply:
Yeah, this is also an option. I'm okay with your suggestion or even having `semantic-release` back as our devDependencies.
@okonet, what's your take on this?

Reply:
I feel this is slightly better because if and when we drop node 4 support(maintenance ends in April 2018), we can switch back to `yarn install`.

Reply:
I agree. I don't like having it in `devDependencies` since it implies it can be run locally which is not what we want.

Reply:
make sense!
I just updated this PR, time to review 🎉